doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -175,45 +135,49 @@ The following code block shows a basic implementation of :c:func:`lte_lc_conn_ev
 
    int main(void)
    {
-   	int err;
+           int err;
+
+           printk("Connecting to LTE network. This may take a few minutes...\n");
 
-   	printk("Connecting to LTE network. This may take a few minutes...\n");
+           err = lte_lc_init_and_connect_async(lte_handler);
+           if (err) {
+                   printk("lte_lc_init_and_connect_async, error: %d\n", err);
+                   return 0;
+           }
 
-   	err = lte_lc_init_and_connect_async(lte_handler);
-   	if (err) {
-   		printk("lte_lc_init_and_connect_async, error: %d\n", err);
-   		return 0;
-   	}
+           k_sem_take(&lte_connected, K_FOREVER);
 
-   	k_sem_take(&lte_connected, K_FOREVER);
+           struct lte_lc_conn_eval_params params = {0};
 
-	struct lte_lc_conn_eval_params params = {0};
+           err = lte_lc_conn_eval_params_get(&params);
+           if (err) {
+                   printk("lte_lc_conn_eval_params_get, error: %d\n", err);
+                   return 0;
+           }
 
-	err = lte_lc_conn_eval_params_get(&params);
-	if (err) {
-		printk("lte_lc_conn_eval_params_get, error: %d\n", err);
-		return 0;
-	}
+           /* Handle connection evaluation parameters... */
 
-	/* Handle connection evaluation parameters... */
-   	/* Continue execution... */
+           /* Continue execution... */
    }
 
 :c:struct:`lte_lc_conn_eval_params` lists all information that is available when performing connection pre-evaluation.
 
-Modem sleep and TAU prewarning notifications
-============================================
+Modem sleep and TAU pre-warning notifications
+=============================================
 
 Modem firmware version 1.3.0 and higher supports receiving callbacks from the modem related to Tracking Area Updates (TAU) and modem sleep.
 Based on these notifications, the application can alter its behavior to optimize for a given metric.
+
 For instance, TAU pre-warning notifications can be used to schedule data transfers prior to a TAU so that data transfer and TAU occurs within the same RRC connection window, thereby saving the potential overhead associated with the additional data exchange.
```

Comment:
```suggestion
For instance, TAU pre-warning notifications can be used to schedule data transfers before a TAU so that data transfer and TAU occurs within the same RRC connection window, thereby saving the potential overhead associated with the additional data exchange.
```